src/lib.rs

```diff
@@ -81,6 +81,7 @@ pub mod input;
 pub mod locale;
 pub mod logging;
 pub mod meter;
+pub mod platform;
```

Comment:
Having a file `platform.rs` is a bit suboptimal in naming and usage. If the idea is to support all platforms, it should be a module platform with `windows.rs` inside. However since this change doesn't include any changes for macos and linux, I'd recommend going a different route.

I think that just renaming this to `panic.rs` should make it super easy for experienced and inexperienced users to figure out what happens in there without ever looking at it. It could be something more specific like `gui_panic.rs` or similar, but I feel like `panic.rs` gives a super nice balance between conciseness and self-description. Only problem would be potential conflicts with std/other crates, so I'm open for better ideas!

Reply:
`panic` seems reasonable.  I have renamed it thereto.